cmd/vulcan-blast-radius/main.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2022 Adevinta
+*/
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	check "github.com/adevinta/vulcan-check-sdk"
+	checkstate "github.com/adevinta/vulcan-check-sdk/state"
+	"github.com/adevinta/vulcan-checks/cmd/vulcan-blast-radius/securitygraph"
+	report "github.com/adevinta/vulcan-report"
+)
+
+var (
+	name            = "vulcan-blast-radius"
+	logger          = check.NewCheckLog(name)
+	blastRadiusVuln = report.Vulnerability{
+		Summary:     "Blast Radius Score: %.2f",
+		Description: "Gives an idea about how many resources are in danger to be compromised if an asset is itself compromised.",
+		Score:       report.SeverityThresholdNone,
+		Recommendations: []string{
+			"The higher the score, the higher the number of resources an asset can potentially grant access to if compromised.",
+			"Try to limit the number of resources that an asset has access to the minimum.",
+			"Pay extra attention to the security of the assets with high blast radius.",
```

Comment:
So it will be like this?: `Pay extra attention to security of assets with high blast radius` ?
It sounds a little bit strange to me I prefer the current one.

Reply:
Sorry, I meant `Pay extra attention to the security of assets with high blast radius.`. I forgot there were 2 "the". But, again, up to you to decide what sounds better. I'm not a native speaker, so... :P